app/src/main/java/com/futurice/freesound/feature/search/DefaultSearchDataModel.java

```diff
@@ -38,7 +38,7 @@
     private final FreeSoundSearchService freeSoundSearchService;
 
     @NonNull
-    private final BehaviorSubject<Option<List<Sound>>> lastResults = BehaviorSubject.create();
+    private final BehaviorSubject<Option<List<Sound>>> lastResultsStream = BehaviorSubject.create();
```

Comment:
OnceAndStream guarantees that there will be value emitted after subscription. So if BS does not have a default value, it cannot be OnceAndStream.
Additionally, when I've changed it to have a default value, multiple tests were failing as this is not the desired behavior